apps/settings/locales/settings.en-US.properties

```diff
@@ -76,7 +76,7 @@ wifi-bad-credentials-confirm=Unable to connect to {{ssid}}, authentication faile
 # in headers, you can use a shorter string here.
 manageNetworks-header={{manageNetworks}}
 manageCertificates=Manage certificates
-manageCert=Manage Certificates
+manageCert=Manage Cert.
```

Comment:
This logical string is not used anywhere together with your fix for https://github.com/harman-red-square/gaia/pull/20/files#diff-f2665f947985d32695435599f3c3600dL5 , so I think we can simply remove this line.